generators/ci-cd/templates/jenkins/_Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env groovy
 
-node {
+node('node') {
```

Comment:
I'm really not sure about that : the default should be to execute on the master. You shouldn't have to provision a node